### ISSUE TITLE
error: cannot find symbol class RNLog

### ISSUE THREAD

**Issue by aliwaqassi** [th3rdwave/react-native-safe-area-context#90]
```
"react": "16.11.0",
"react-native": "0.62.2",
```

**Comment by mikehardy** [comment]
@janicduplessis this symbol doesn't exist until react-native 0.63: https://github.com/facebook/react-native/commit/52b3105f652eca72892f200923e1687f1d995486#diff-f56e2273ec5976b7dd639a213b0f9cd4

This breaks the build on android

**Comment by mikehardy** [comment]
module version 3.0.0

```
/Users/mike/work/Kullki/ksocialscore/packages/public-app/node_modules/react-native-safe-area-context/android/src/main/java/com/th3rdwave/safeareacontext/SafeAreaView.java:12: error: cannot find symbol
import com.facebook.react.util.RNLog;
                              ^
  symbol:   class RNLog
  location: package com.facebook.react.util
/Users/mike/work/Kullki/ksocialscore/packages/public-app/node_modules/react-native-safe-area-context/android/src/main/java/com/th3rdwave/safeareacontext/SafeAreaView.java:91: error: cannot find symbol
        RNLog.w(getReactContext(this), "SafeAreaView: Timed out waiting for layout.");
        ^
  symbol:   variable RNLog
  location: class SafeAreaView
2 errors
```

**Comment by mikehardy** [comment]
apparently fixed in 3.0.2 :-)
